aidial_adapter_vertexai/llm/consumer.py

```diff
@@ -0,0 +1,53 @@
+from abc import ABC, abstractmethod
+from typing import Callable, Coroutine, Optional
+
+from aidial_sdk.chat_completion import Choice
+
+from aidial_adapter_vertexai.universal_api.token_usage import TokenUsage
+
+
+class Consumer(ABC):
+    @abstractmethod
+    async def append_content(self, content: str):
+        pass
+
+    @abstractmethod
+    async def add_usage(self, usage: TokenUsage):
```

Comment:
Maybe set_usage(), given that it's supposed to be called once?

Reply:
Fixed